eth/protocols/eth/protocol.go

```diff
@@ -224,22 +224,21 @@ type BlockBody struct {
 	Transactions []*types.Transaction // Transactions contained within a block
 	Uncles       []*types.Header      // Uncles contained within a block
 	Withdrawals  []*types.Withdrawal  `rlp:"optional"` // Withdrawals contained within a block
-	Requests     []*types.Request     `rlp:"optional"` // Requests contained within a block
+	Requests     [][]byte             `rlp:"optional"` // Requests contained within a block
```

Comment:
I just made a PR to remove it

https://github.com/ethereum/go-ethereum/pull/30562